test-network-nano-bash/README.md

```diff
@@ -0,0 +1,78 @@
+
+# Prereqs
+
+- Follow the Fabric documentation for the [Prereqs](https://hyperledger-fabric.readthedocs.io/en/latest/prereqs.html)
+- Follow the Fabric documentation for [downloading the Fabric samples and binaries](https://hyperledger-fabric.readthedocs.io/en/latest/install.html). You can skip the docker image downloads by using `curl -sSL https://bit.ly/2ysbOFE | bash -s -- -d`
+
+# Instructions for starting network
+
+Open terminal windows for 3 ordering nodes, 4 peer nodes, and 4 peer admins as seen in the following terminal setup.
+Note, you can start with two ordering nodes and single peer node and single peer admin terminal if you would like to keep things even more minimal (two ordering nodes are required to achieve consensus (2 of 3), while a single peer can be utilized since the endorsement policy is set as any single organization).
+![Terminal setup](terminal_setup.png)
+
+
+- cd to the `test-network-nano-bash` directory in each terminal window
+- In the first orderer terminal, run `./generate_artifacts.sh` to generate crypto material (calls cryptogen) and system and application channel genesis block and configuration transactions (calls configtxgen)
+- In the three orderer terminals, run `./orderer1.sh`, `./orderer2.sh`, `./orderer3.sh` respectively
+- In the four peer terminals, run `./peer1.sh`, `./peer2.sh`, `./peer3.sh`, `./peer4.sh` respectively
+- In the four peer admin terminals, run `. peer1admin.sh`, `. peer2admin.sh`, `. peer3admin.sh`, `. peer4admin.sh` respectively
+
+Note the syntax of running the scripts. The peer admin scripts run with the dot command in order to source the script files in the respective shells. This is important so that the exported environment variables can be utilized by any subsequent user commands.
+
+The `peer1admin.sh` script sets the peer1 admin environment variables, creates the application channel `mychannel`, updates the channel configuration for the org1 gossip anchor peer, and joins peer1 to `mychannel`.
+The remaining peer admin scripts join their respective peers to `mychannel`.
+
+# Instructions for deploying and running the basic asset transfer sample chaincode
+
+Package and install the chaincode on peer1:
+
+```
+peer lifecycle chaincode package basic.tar.gz --path ../asset-transfer-basic/chaincode-go --lang golang --label basic_1
+
+peer lifecycle chaincode install basic.tar.gz
+```
+
+The chaincode install may take a minute since the `fabric-ccenv` chaincode builder docker image will be downloaded if not already available on your machine.
+Copy the returned chaincode package ID into an environment variable for use in subsequent commands (your ID may be different):
+
+```
+CC_PACKAGE_ID=basic_1:faaa38f2fc913c8344986a7d1617d21f6c97bc8d85ee0a489c90020cd57af4a5
+```
+
+Approve and commit the chaincode (only a single approver is required based on the lifecycle endorsement policy of any organization):
```

Comment:
Is the test-network the same?  From memory I thought that the test-network had to have both orgs approve the chaincode definition.

Reply:
You're right, I've clarified this in the README now. I prefer to have a single endorser in this nano sample since it will be used by Fabric developers to iterate quickly on deployment mechanics.